.github/workflows/config/spelling_allowlist.txt

```diff
@@ -214,19 +224,27 @@ lossy
 lvalue
 makefiles
 merchantability
+mgpu
 multithreaded
 mutex
 namespace
 namespaces
 natively
 normalization
+np
 nullary
+nvidia
+occa
+occb
+openfermion
 optimizer
 optimizers
 parallelization
 parallelize
 parallelizing
+param
 parameterization
+params
```

Comment:
I don't think `param` and `params` are words and therefore should probably not be added to the allowlist.